### ISSUE TITLE
tracee single binary should not load plugin if traced events is not a signature event

### ISSUE THREAD

**Issue by rafaeldtinoco** [aquasecurity/tracee#3070]
## Description

After discussions at https://github.com/aquasecurity/tracee/discussions/3065#discussioncomment-5837732 (not necessiraly related to the issue being described here) and to check the following bug: https://github.com/aquasecurity/tracee/issues/2778 we should, perhaps, seek a way of NOT loading the plugin mechanism if events being traced aren't signature events.

This could avoid errors when trying to execute the `tracee` single binary when compiled statically, like described at https://github.com/aquasecurity/tracee/discussions/3065#discussioncomment-5837634

**Comment by rafaeldtinoco** [comment]
@josedonizetti thoughts ?

**Comment by josedonizetti** [comment]
@rafaeldtinoco 
yes, 100% , when I was debugging the deadlocks, I also saw we start the engine in the pipeline, even when it is not needed, and we already have a flag to cover it there -> https://github.com/aquasecurity/tracee/blob/main/pkg/ebpf/events_pipeline.go#L66, we just need to fix the engine creation to not enable the engine if no events signature events are passed -> https://github.com/aquasecurity/tracee/blob/main/pkg/cmd/cobra/cobra.go#L251

I can take this issue  if you busy with other tasks.

**Comment by rafaeldtinoco** [comment]
@josedonizetti all yours if and when you have time. Thanks :*.
